proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and accessing OCI Runtime and Image specification JSON.
+
+## Proposal
+With repositories under the https://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/giuseppe/libocispec to https://github.com/opencontainers/libocispec.
+
+### Initial Maintainers
```

Comment:
Is there a reason why you are proposing maintainers here that have not contributed code or issues directly to the project? 

https://github.com/giuseppe/libocispec/graphs/contributors

Reply:
at the time I proposed libocispec I asked around if someone was interested in co-maintaining it so to not have only one maintainer listed.  I'll update the list if this is not the preferred way of doing it.